frontend/src/components/Pf/PfColors.tsx

```diff
@@ -127,43 +131,54 @@ export type PFColorValues = {
 
 export let PFColorVals: PFColorValues;
 
+/*
+  Subtract color from var
```

Comment:
```suggestion
  Extract color from var
```

Reply:
Shit I marked this like resolved I'll do the change thanks